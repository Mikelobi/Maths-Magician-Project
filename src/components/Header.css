.header-title {
  padding-left: 1em;
  font-size: 2em;
  letter-spacing: 0.05em;
}
