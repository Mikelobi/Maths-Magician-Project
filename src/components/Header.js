import React from 'react';
import './Header.css';

const Header = () => (
  <div>
    <h1 className="header-title">Math Magicians</h1>
  </div>
);

export default Header;
