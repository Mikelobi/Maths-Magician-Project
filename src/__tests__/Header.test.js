import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Test presence of Quotes component', () => {
  test('renders the Quotes component correctly', () => {
    const navComponent = renderer
      .create(
        <BrowserRouter>
          <Header />
        </BrowserRouter>,
      )
      .toJSON();
    expect(navComponent).toMatchSnapshot();
  });
});
