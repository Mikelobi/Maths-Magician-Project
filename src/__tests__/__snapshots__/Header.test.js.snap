// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test presence of Quotes component renders the Quotes component correctly 1`] = `
<div>
  <h1
    className="header-title"
  >
    Math Magicians
  </h1>
</div>
`;
